static/app/views/settings/project/projectUserFeedback.tsx

```diff
@@ -86,22 +100,16 @@ class ProjectUserFeedbackSettings extends AsyncView<Props> {
             the issue in Sentry.`
           )}
         </TextBlock>
-        <ButtonList>
-          <Button external href="https://docs.sentry.io/product/user-feedback/">
-            {t('Read the docs')}
-          </Button>
-          <Button priority="primary" onClick={this.handleClick}>
-            {t('Open the report dialog')}
-          </Button>
-        </ButtonList>
+
+        <PermissionAlert access={['project:write']} project={project} />
```

Comment:
this is below the text block?
<img width="1089" alt="Screenshot 2023-04-28 at 14 37 01" src="https://user-images.githubusercontent.com/70817427/235258497-e9630d14-5a4e-46e8-962e-ed29cc5bf5d7.png">

Reply:
```suggestion

        <PermissionAlert project={project} />
```
also the default `access` in PermissionAlert is `['project:write']`